base/dotenv.default.mustache

```diff
@@ -0,0 +1,3 @@
+PORT=
+RETHINK_HOST=
+RETHINK_NAME=
```

Comment:
It is `.gitignore`d, but I like to take an onion approach with these things. The more tools we can use to stop testing data ending up in prod the better.

It sounds like we've reached an accord on the fact that there's no need for three files. We're now shooting for whether we can actually take two files down to one, correct? I think that is a reasonable question to ask, and one to which the answer may well be "Yeah, actually that would be good!", but we should not let it hold us up moving to dotenv-safe. If we're happy, let's just get this using the example then pull the trigger on this PR.